app/middlewares/api_request_middleware.rb

```diff
@@ -29,7 +29,7 @@ def _call(env)
 
     begin
       if trace_request?
-        ApiRequestJob.perform_async(request_data.stringify_keys, response_data.stringify_keys, status, Time.zone.now.to_s)
+        ApiRequestJob.perform_async(request_data.stringify_keys, response_data.stringify_keys, status, Time.zone.now.to_s, RequestLocals.fetch(:dfe_analytics_request_id) { nil })
```

Comment:
It took me a minute to figure out why we're explicitly setting/passing around the analytics request ID; worth adding a comment around this?